tests/TestCase/Controller/SitemapsControllerTest.php

```diff
@@ -103,4 +103,10 @@ public function testIndexWithModels() {
 
 		$Controller->index();
 	}
+
+	public function testIndexAccess() {
```

Comment:
Docblock here please something along the lines of:

/**
 * Test the index method that the correct route is loaded and works. Essentially
 * this test validates the routes.php file as being correct and working.
 *
 * @return void
 */